src/redux/books/books.js

```diff
@@ -19,8 +19,8 @@ export const addBook = (book) => ({
   type: ADD, book,
 });
 
-export const removeBook = (id) => ({
-  type: REMOVE, id,
+export const removeBook = (book) => ({
+  type: REMOVE, book,
```

Comment:
[OPTIONAL]

- I think it will be better if you move your actions up before the reducer to make your code cleaner 👍🏽